client/components/tinymce/plugins/wpcom/plugin.js

```diff
@@ -32,7 +32,7 @@ function wpcomPlugin( editor ) {
 
 		if ( event.content ) {
 			if ( event.content.indexOf( '<!--more' ) !== -1 ) {
-				title = __( 'Read more...' );
```

Comment:
>While we're on the topic, I'd like to check that using `i18n-calypso` makes sense from the perspective of someone with more investment in TinyMCE. `i18n-calypso` seems easier to me, but  I'm not sure what our reasons are/were for including `i18n.js`, and I wouldn't want to unilaterally reverse a thought out decision.

Related: 8742-gh-calypso-pre-oss

My memory is fuzzy, so I'm not exactly sure why we chose to create the separate `i18n.js` file instead of calling on `i18n-calypso` directly. One possible explanation is that it could simplify any upstream WordPress patches or plugins we'd want to make? I dunno, more I look at it, more it seems we should remove `i18n.js` altogether since it adds complexity to the `<TinyMCE />` component too (separate language initialization, workarounds to avoid requesting a localization file).